src/components/Game.jsx

```diff
@@ -32,7 +32,11 @@ class Game extends Component {
   }
 
   calculateScore(word) {
-    return word.length
+    if (word.length >= 3 && word.length <= 8) {
```

Comment:
Im still working on this. Will fix when the word length is 1. The score should be 0.